packages/playground/src/components/user-management/password-copy-action.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react'
+import { Text, Icon, cn } from '@harnessio/canary'
+import copy from 'clipboard-copy'
+import { ShaBadge } from '../..'
+
+export const PasswordCopyAction = ({ password }: { password: string }) => {
```

Comment:
is this styled differently than commitactions button?

Reply:
We don't have this design on Figma right now. This is the design in the orignal gitness. I've added more props for copying password here so might not affect other pages using the commitActions. Please check it out!

<img width="791" alt="Screenshot 2024-10-29 at 11 43 53 AM" src="https://github.com/user-attachments/assets/be71a12e-584d-4670-8051-f5ff30abe0cd">

@cjlee01